dot/types/body.go

```diff
@@ -95,6 +95,17 @@ func NewBodyFromExtrinsicStrings(ss []string) (*Body, error) {
 	return &body, nil
 }
 
+func (b *Body) String() string {
+	fmt.Println("body.String")
+	if b == nil {
+		return "nil"
```

Comment:
why not return an empty string?

Reply:
yeah I'll update this. this was updated since I was debugging a test